CHANGELOG.md

```diff
@@ -3315,6 +3315,11 @@ This is the last release to support building with Swift 4.0 and Swift 4.1.
 
 #### Enhancements
 
+* Add `period_spacing` opt-in rule that checks periods are not followed
```

Comment:
Can you move this to the topmost section? This is in the release notes section for the 0.26 release from 2018 👴

Reply:
🤦 Sure thing!